chat/command.go

```diff
@@ -212,16 +209,17 @@ func InitCommands(c *Commands) {
 				if cfg.Theme != nil {
 					theme = cfg.Theme.ID()
 				}
-				var output bytes.Buffer
-				fmt.Fprintf(&output, "Current theme: %s%s", theme, message.Newline)
-				fmt.Fprintf(&output, "   Themes available: ")
+				var output string
```

Comment:
Why is it better to accumulate by replacing an immutable string repeatedly instead of writing into a buffer?

Reply:
It makes the code cleaner and easier to understand, imo speed in this case never matters since its just showing the themes available.

Reply:
I'd be happy with string.Builder (it didn't exist when I wrote this). Mind changing that?

It's not so much about speed as it is about avoiding allocations in a loop (avoiding GC pressure) and consistency with the rest of the codebase.

Reply:
Done :)